kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProduceRateLimiters.java

```diff
@@ -66,14 +66,14 @@ public void rateLimit(String clusterId, long requestSize) {
     if (!rateLimitingEnabled) {
       return;
     }
-
+    // Global rate limit first to reduce CPU usage under load
+    // https://confluentinc.atlassian.net/browse/KREST-4979
```

Comment:
super-nit: In other places we are using just the KREST-XXX format: https://github.com/confluentinc/kafka-rest/search?q=KREST. Having said that though I just realized this comment can be linked to the JIRA anyway via `git blame` because you annotate commits just like me, so if you want just disregard my suggestion.